uptane/clients/primary.py

```diff
@@ -1154,13 +1159,22 @@ def validate_time_attestation(self, timeserver_attestation):
     # Extract actual time from the timeserver's signed attestation.
     new_timeserver_time = timeserver_attestation['signed']['time']
 
+    # Make sure the format is understandable to us before saving the
+    # attestation and time.  Convert to a UNIX timestamp.
+    new_timeserver_time_unix = int(tuf.formats.datetime_to_unix_timestamp(
+        iso8601.parse_date(new_timeserver_time)))
+    tuf.formats.UNIX_TIMESTAMP_SCHEMA.check_match(new_timeserver_time_unix)
```

Comment:
Stupid question but shouldn't `datetime_to_unix_timestamp` be guaranteed to return a unix timestamp (if it doesn't raise an exception)?

Reply:
It seems to always return a unix timestamp unless it raises an error, but it's still code sitting in a different project, so I felt like a defensive check might be wise.